platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -0,0 +1,648 @@
+/* Copyright (c) 2014-2018, Linaro Limited
+ * Copyright (c) 2021, ARM Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+#include <odp_posix_extensions.h>
+#include <odp/api/crypto.h>
+#include <odp_init_internal.h>
+#include <odp/api/spinlock.h>
+#include <odp/api/sync.h>
+#include <odp/api/debug.h>
+#include <odp/api/align.h>
+#include <odp/api/shared_memory.h>
+#include <odp_debug_internal.h>
+#include <odp/api/hints.h>
+#include <odp/api/random.h>
+#include <odp/api/plat/packet_inlines.h>
+#include <odp/api/plat/thread_inlines.h>
+#include <odp_packet_internal.h>
+#include <odp/api/plat/queue_inlines.h>
+
+#include "AArch64cryptolib.h"
+
+/* Inlined API functions */
+#include <odp/api/plat/event_inlines.h>
+
+#define MAX_SESSIONS 4000
+/* Length in bytes */
+#define _ARM_CRYPTO_MAX_CIPHER_KEY_LENGTH      64
+#define _ARM_CRYPTO_MAX_AUTH_KEY_LENGTH        64
+#define _ARM_CRYPTO_MAX_IV_LENGTH              16
```

Comment:
I do not think these should start with underscore.

Reply:
Sure, I will remove the underscore. Thanks!